project/src/Controller/SiteController.php

```diff
@@ -136,4 +140,23 @@ public function removeDomainName(Site $site, DomainName $domainName, EntityManag
         ]);
     }
 
+    /**
+     * @param Collection $siteClientToServicesBinders
+     * @param EntityManagerInterface $entityManager
+     * @return array[]
+     */
+    private function getServices(Collection $siteClientToServicesBinders, EntityManagerInterface $entityManager): array
```

Comment:
This function is the same everywhere ?
If yes you can put this function in a trait for example : https://www.php.net/manual/fr/language.oop5.traits.php
Or in a static class that you can call everywhere like this `MyClass::getServices` (I prefer this method)